src/test/java/org/killbill/billing/plugin/dwolla/TestDwollaClient.java

```diff
@@ -0,0 +1,93 @@
+package org.killbill.billing.plugin.dwolla;
+
+import com.dwolla.java.sdk.responses.TokenResponse;
+import io.swagger.client.ApiClient;
+import io.swagger.client.ApiException;
+import io.swagger.client.api.AccountsApi;
+import io.swagger.client.api.CustomersApi;
+import io.swagger.client.api.FundingsourcesApi;
+import io.swagger.client.api.TransfersApi;
+import io.swagger.client.auth.ApiKeyAuth;
+import io.swagger.client.model.*;
+import org.killbill.billing.plugin.dwolla.client.DwollaClient;
+import org.killbill.billing.plugin.dwolla.client.DwollaConfigProperties;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.Properties;
+
+public class TestDwollaClient {
+
+    private static final String PROPERTY_PREFIX = "org.killbill.billing.plugin.dwolla.";
+
+    public final static String SCOPES = "Send|AccountInfoFull|Funding";
+    public final static String REDIRECT_URI = "http://localhost:4567/callback";
+    public final static String CLIENT_ID = "SC9t2DfUDxx0JFI80y9DeOL1ZduXEEXZbC9qCdJDr5XGhfLKcr";
```

Comment:
Probably want to abstract this in a properties file that is not checked-in (Adyen does it too).